external-crates/move/crates/move-vm-integration-tests/src/tests/loader_tests.rs

```diff
@@ -349,6 +355,20 @@ impl LinkageResolver for RelinkingStore {
             .unwrap_or(module_id)
             .clone())
     }
+
+    fn all_package_dependencies(&self) -> Result<BTreeSet<AccountAddress>, Self::Error> {
```

Comment:
A bit of confusion: it appears we also define `relinking_store.rs` below, which could b emoved to `move-vm-test-utils/src/storage.rs` If we did that, could we also reuse it here?